router.go

```diff
@@ -15,17 +15,19 @@ import (
 	"github.com/checkmarble/marble-backend/utils"
 )
 
-func corsOption(env string) cors.Config {
-	allowedOrigins := []string{"https://app*.checkmarble.com", "https://backoffice*.checkmarble.com"}
+func corsOption(conf AppConfiguration) cors.Config {
+	allowedOrigins := []string{
+		"https://" + conf.config.MarbleAppHost,
+		"https://" + conf.config.MarbleBackofficeHost,
+	}
```

Comment:
I would go for something like that :
```go
protocol := "https://"
if conf.env == "development" {
  protocol = "http://"
}

allowedOrigins := []string{
  protocol + conf.config.MarbleAppHost,
  protocol + conf.config.MarbleBackofficeHost,
}
```

Reply:
ah, yes pretty easy to do here, I'll do that